input_size: 17
input_ids: ['object_pos', 'joints_pos','joints_vel'] #['last_ee_pos', 'joints_pos','joints_vel']
output_ids: ['command_vel']
hidden_size1: 64
hidden_size2: 64
output_size: 7
lr: 0.001
epochs: 10000
batch_size: 32
load_dir: /home/adriano/Desktop/canopies/code/CanopiesSimulatorROS/workspace/src/imitation_learning/data
save_dir: /home/adriano/Desktop/behavioural_cloning/params #/home/adriano/Desktop/canopies/code/CanopiesSimulatorROS/workspace/src/imitation_learning/src/agent_params
task: grasp2 #push
device: cuda
#